partner_credit_limit/tests/common.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+# Copyright 2016 Vauxoo (https://www.vauxoo.com) <info@vauxoo.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from openerp.tests.common import TransactionCase
+from openerp import tools
+from openerp.modules.module import get_module_resource
+
+
+class TestCommon(TransactionCase):
+
+    def setUp(self):
+        super(TestCommon, self).setUp()
+        self.account_invoice = self.env['account.invoice']
+        self.account_invoice_line = self.env['account.invoice.line']
+
+    def _load(self, module, *args):
```

Comment:
This code copied it from [here](https://github.com/odoo/odoo/blob/9.0/addons/account_asset/tests/test_account_asset.py#L10), I needed accounting data demo that in version 9 is no already loaded, but there are the files with the information.